requirements.txt

```diff
@@ -1,6 +1,7 @@
 # setup.py install_requires
 # any changes here should also be reflected in setup.py "install_requires"
 boto3 ~= 1.20
+getdaft==0.1.11
```

Comment:
Should this now be `getdaft==0.1.12`? Also, I think we can remove the corresponding line to install daft from https://github.com/ray-project/deltacat/blob/main/benchmark-requirements.txt#L4C11-L4C11 since it should be transitively included once this change is merged.

Reply:
Done, and done!